html-block-and-inline/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Block and Inline</title>
+</head>
+
+<body>
+  <h1>Blue Americana Sunggie&reg; Blanket</h1>
+  <h2>by SunggieStore</h2>
+  <p>Price: <strike> $19.99 </strike> $14.99</p>
+  <p>Part of our new Americana collection, this <em>beautiful</em> blue is already a customer favorite!</p>
+  <p>Enjoy complete <b>Freedon of Movement!</b></p>
+  <div>Does not slip and slide like a regulare blanket nd keeps you warm from head to toe.</div>
```

Comment:
You shouldn't use `div` elements as text elements, this is better suited to be a `p` element.